diff --git a/node_modules/@react-navigation/elements/lib/module/Screen.js b/node_modules/@react-navigation/elements/lib/module/Screen.js
index e4ff174..38ed0a7 100644
--- a/node_modules/@react-navigation/elements/lib/module/Screen.js
+++ b/node_modules/@react-navigation/elements/lib/module/Screen.js
@@ -52,7 +52,6 @@ export function Screen(props) {
         }] : null],
         children: /*#__PURE__*/_jsx(View, {
           ref: headerRef,
-          pointerEvents: "box-none",
           onLayout: e => {
             const {
               height
@@ -82,7 +81,8 @@ const styles = StyleSheet.create({
     flex: 1
   },
   header: {
-    zIndex: 1
+    zIndex: 1,
+    pointerEvents: 'box-none'
   },
   absolute: {
     position: 'absolute',
diff --git a/node_modules/@react-navigation/elements/src/Header/Header.tsx b/node_modules/@react-navigation/elements/src/Header/Header.tsx
index ff00094..48bebcb 100644
--- a/node_modules/@react-navigation/elements/src/Header/Header.tsx
+++ b/node_modules/@react-navigation/elements/src/Header/Header.tsx
@@ -309,32 +309,45 @@ export function Header(props: Props) {
 
   return (
     <Animated.View
-      pointerEvents="box-none"
-      style={[{ height, minHeight, maxHeight, opacity, transform }]}
+      style={[
+        {
+          pointerEvents: 'box-none',
+          height,
+          minHeight,
+          maxHeight,
+          opacity,
+          transform,
+        },
+      ]}
     >
       <Animated.View
-        pointerEvents="box-none"
-        style={[StyleSheet.absoluteFill, backgroundContainerStyle]}
+        style={[
+          StyleSheet.absoluteFill,
+          { pointerEvents: 'box-none' },
+          backgroundContainerStyle,
+        ]}
       >
         {headerBackground ? (
           headerBackground({ style: backgroundStyle })
         ) : (
           <HeaderBackground
-            pointerEvents={
-              // Allow touch through the header when background color is transparent
-              headerTransparent &&
-              (backgroundStyle.backgroundColor === 'transparent' ||
-                Color(backgroundStyle.backgroundColor).alpha() === 0)
-                ? 'none'
-                : 'auto'
-            }
-            style={backgroundStyle}
+            style={[
+              {
+                // Allow touch through the header when background color is transparent
+                pointerEvents:
+                  headerTransparent &&
+                  (backgroundStyle.backgroundColor === 'transparent' ||
+                    Color(backgroundStyle.backgroundColor).alpha() === 0)
+                    ? 'none'
+                    : 'auto',
+              },
+              backgroundStyle,
+            ]}
           />
         )}
       </Animated.View>
-      <View pointerEvents="none" style={{ height: headerStatusBarHeight }} />
+      <View style={{ pointerEvents: 'none', height: headerStatusBarHeight }} />
       <View
-        pointerEvents="box-none"
         style={[
           styles.content,
           Platform.OS === 'ios' && frame.width >= IPAD_MINI_MEDIUM_WIDTH
@@ -343,7 +356,6 @@ export function Header(props: Props) {
         ]}
       >
         <Animated.View
-          pointerEvents="box-none"
           style={[
             styles.start,
             !searchBarVisible && headerTitleAlign === 'center' && styles.expand,
@@ -356,7 +368,6 @@ export function Header(props: Props) {
         {Platform.OS === 'ios' || !searchBarVisible ? (
           <>
             <Animated.View
-              pointerEvents="box-none"
               style={[
                 styles.title,
                 {
@@ -393,7 +404,6 @@ export function Header(props: Props) {
               })}
             </Animated.View>
             <Animated.View
-              pointerEvents="box-none"
               style={[
                 styles.end,
                 styles.expand,
@@ -445,6 +455,7 @@ export function Header(props: Props) {
 
 const styles = StyleSheet.create({
   content: {
+    pointerEvents: 'box-none',
     flex: 1,
     flexDirection: 'row',
     alignItems: 'stretch',
@@ -454,16 +465,19 @@ const styles = StyleSheet.create({
   },
   title: {
     justifyContent: 'center',
+    pointerEvents: 'box-none',
   },
   start: {
     flexDirection: 'row',
     alignItems: 'center',
     justifyContent: 'flex-start',
+    pointerEvents: 'box-none',
   },
   end: {
     flexDirection: 'row',
     alignItems: 'center',
     justifyContent: 'flex-end',
+    pointerEvents: 'box-none',
   },
   expand: {
     flexGrow: 1,
diff --git a/node_modules/@react-navigation/elements/src/Header/HeaderSearchBar.tsx b/node_modules/@react-navigation/elements/src/Header/HeaderSearchBar.tsx
index 9310f86..65fac2b 100644
--- a/node_modules/@react-navigation/elements/src/Header/HeaderSearchBar.tsx
+++ b/node_modules/@react-navigation/elements/src/Header/HeaderSearchBar.tsx
@@ -157,10 +157,16 @@ function HeaderSearchBarInternal(
 
   return (
     <Animated.View
-      pointerEvents={visible ? 'auto' : 'none'}
       aria-live="polite"
       aria-hidden={!visible}
-      style={[styles.container, { opacity: visibleAnim }, style]}
+      style={[
+        styles.container,
+        {
+          pointerEvents: visible ? 'auto' : 'none',
+          opacity: visibleAnim,
+        },
+        style,
+      ]}
     >
       <View style={styles.searchbarContainer}>
         <HeaderIcon
diff --git a/node_modules/@react-navigation/elements/src/ResourceSavingView.tsx b/node_modules/@react-navigation/elements/src/ResourceSavingView.tsx
index 63e1dc0..6d7ec56 100644
--- a/node_modules/@react-navigation/elements/src/ResourceSavingView.tsx
+++ b/node_modules/@react-navigation/elements/src/ResourceSavingView.tsx
@@ -27,11 +27,13 @@ export function ResourceSavingView({
         // @ts-expect-error: hidden exists on web, but not in React Native
         hidden={!visible}
         style={[
-          { display: visible ? 'flex' : 'none' },
+          {
+            display: visible ? 'flex' : 'none',
+            pointerEvents: visible ? 'auto' : 'none',
+          },
           styles.container,
           style,
         ]}
-        pointerEvents={visible ? 'auto' : 'none'}
         {...rest}
       >
         {children}
@@ -40,11 +42,7 @@ export function ResourceSavingView({
   }
 
   return (
-    <View
-      style={[styles.container, style]}
-      // box-none doesn't seem to work properly on Android
-      pointerEvents={visible ? 'auto' : 'none'}
-    >
+    <View style={[styles.container, style]}>
       <View
         collapsable={false}
         removeClippedSubviews={
@@ -52,7 +50,6 @@ export function ResourceSavingView({
           // This is an workaround for a bug where the clipped view never re-appears
           Platform.OS === 'ios' || Platform.OS === 'macos' ? !visible : true
         }
-        pointerEvents={visible ? 'auto' : 'none'}
         style={visible ? styles.attached : styles.detached}
       >
         {children}
@@ -68,9 +65,11 @@ const styles = StyleSheet.create({
   },
   attached: {
     flex: 1,
+    pointerEvents: 'auto',
   },
   detached: {
     flex: 1,
     top: FAR_FAR_AWAY,
+    pointerEvents: 'none',
   },
 });
diff --git a/node_modules/@react-navigation/elements/src/Screen.tsx b/node_modules/@react-navigation/elements/src/Screen.tsx
index 68455b1..4b2672f 100644
--- a/node_modules/@react-navigation/elements/src/Screen.tsx
+++ b/node_modules/@react-navigation/elements/src/Screen.tsx
@@ -90,7 +90,6 @@ export function Screen(props: Props) {
           >
             <View
               ref={headerRef}
-              pointerEvents="box-none"
               onLayout={(e) => {
                 const { height } = e.nativeEvent.layout;
 
@@ -126,6 +125,7 @@ const styles = StyleSheet.create({
   },
   header: {
     zIndex: 1,
+    pointerEvents: 'box-none',
   },
   absolute: {
     position: 'absolute',
